frontend/src/index.js

```diff
@@ -12,7 +12,7 @@ import store, { history } from "./store/store";
 
 ReactDOM.render(
     <Provider store={store}>
-        <App history={history} />
+        <App history={history} me={store.getState().auth.me} />
```

Comment:
Oh, you use a different way to get 'me' from the way I use in #71 PR. Could you check the PR and let me know what you think better?

Reply:
I will delete it. Thank you!